yt/yt/ytlib/api/native/client_authentication_impl.cpp

```diff
@@ -117,16 +127,22 @@ TIssueTokenResult TClient::DoIssueTokenImpl(
     const TIssueTokenOptions& options)
 {
     auto tokenHash = GetSha256HexDigestLowerCase(token);
+    auto tokenPrefix = token.substr(0, TokenPrefixLength);
+    if (!tokenPrefix.StartsWith("ytct-")) {
+        tokenPrefix = "";
```

Comment:
I'd prefer an error if it is not empty and not starts with ytct-

Reply:
We have a new API `IssueSpecificToken` that may request a specific token that may not start with `ytct-`

Current QT/YQL logic is:
1. Request a random token from cluster1 -> get TOKEN
2. Request specific token TOKEN from cluster2
3. Request specific token TOKEN from cluster3
4. ...
5. Use TOKEN to execute YQL query that affects multiple clusters

If cluster2, cluster3 get this logic before cluster1, then this process will never succeed and queries will fail to execute